modules/node-observability-scripting.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-observability-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="node-observability-scripting_{context}"]
+= Node Observability Operator Scripting
+
+[About Scripting]
+
+.Prerequisites
+
+* You have installed the Node Observability Operator.
+* You have created the `NodeObservability` custom resource (CR).
+* You have access to the cluster with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `NodeObservabilityScripting` resource file named `nodeobservabilityrun-script.yaml` that contains the following text:
++
+[source,yaml]
+----
+apiVersion: nodeobservability.olm.openshift.io/v1alpha2
+kind: NodeObservabilityRun
+metadata:
+  name: nodeobservabilityrun-script
+  namespace: node-observability-operator
+spec:
+  nodeObservabilityRef:
+    name: cluster
+    type: "scripting"
+----
+
+. Build a container using the `Docker.dev` file.
+
+[IMPORTANT]
+====
+You can request only the following two scripts:
+* metrics.sh
+* network-metrics.sh (uses monitor.sh)
+====
+
+. Use the following command to trigger the scripting by running the `NodeObservabilityRun` resource:
++
+[source,terminal]
+----
+$ oc apply -f nodeobservabilityrun-script.yaml
+----
+
+. Review the status of the `NodeObservabilityRun` by running the following command:
++
+[source,terminal]
+----
+$ oc get nodeobservabilityrun nodeobservabilityrun-script -o yaml  | yq '.status.conditions'
+----
+
++
+.Example output
+[source,terminal]
+----
+Status:
+  Agents:
+    Ip:    10.128.2.252
+    Name:  node-observability-agent-n2fpm
+    Port:  8443
+    Ip:    10.131.0.186
+    Name:  node-observability-agent-wcc8p
+    Port:  8443
+  Conditions:
+    Conditions:
+      Last Transition Time:  2023-12-19T15:10:51Z
+      Message:               Ready to start profiling
+      Reason:                Ready
+      Status:                True
+      Type:                  Ready
+      Last Transition Time:  2023-12-19T15:11:01Z
+      Message:               Profiling query done
+      Reason:                Finished
+      Status:                True
+      Type:                  Finished
+  Finished Timestamp:        2023-12-19T15:11:01Z
+  Start Timestamp:           2023-12-19T15:10:51Z
+----
+
++
+The scripting is complete once the status is `True` and type is `Finished`.
+
+. Retrieve the scripting data from the container's `/run/node-observability` path by running the following bash script:
```

Comment:
Also could you change  `/run/node-observability` to  '/' (root) path

Reply:
Ack.

Reply:
Done